pdns/logger.hh

```diff
@@ -60,6 +60,10 @@ public:
     d_disableSyslog = d;
   }
 
+  void disableTimeStamps(bool d) {
+    d_disableTimeStamps = d;
```

Comment:
the setting is singular but this bool and method are plural, looks a bit weird. I suggest making the setting plural.